src/components/RightPane/SectionTable/SectionTableButtons.tsx

```diff
@@ -109,6 +128,9 @@ export const ScheduleAddCell = withStyles(styles)((props: ScheduleAddCellProps)
                     <MenuItem onClick={() => closeAndAddCourse(scheduleNames.length, true)}>
```

Comment:
Same idea as the other event handler. Is it possible to make `closeAndAddCourse` use the correct value internally, or write a handler function that will? I just try to avoid inlining function calls if I can
```suggestion
                    <MenuItem onClick={closeAndAddCourse}>
```

Reply:
I didn't write it but I test it I can't just use `<MenuItem onClick={closeAndAddCourse}>`. I wrote a handler function to call the `closeAndAddCourse()`, like that ` <MenuItem onClick={addCourseHandler}>`.  It is working.